server/services/referralService.ts

```diff
@@ -20,6 +33,44 @@ export default class ReferralService {
     return referralClient.create(courseOfferingId, prisonNumber, referrerId)
   }
 
+  async getCaseList(
+    username: Express.User['username'],
+    organisationId: Organisation['id'],
+    caseloads: Array<Caseload>,
+  ): Promise<Array<ReferralSummaryDetails>> {
+    const hmppsAuthClient = this.hmppsAuthClientBuilder()
+    const systemToken = await hmppsAuthClient.getSystemClientToken(username)
+    const referralClient = this.referralClientBuilder(systemToken)
+
+    try {
+      const dashboardResponse = await referralClient.dashboard(organisationId)
```

Comment:
if we rename the client method, this can be `const paginatedReferralSummaries = ...`